docs/blog/write-the-docs-2023-stats.rst

```diff
@@ -0,0 +1,74 @@
+.. post:: Jan 10, 2024
+   :tags: stats, year in review
+   :author: Eric Holscher
+
+Write the Docs 2023 Stats
+=========================
+
+We publish a longer :doc:`/blog/2024-january-update` along with our stats, which has more context and information about where we're headed.
+
+2023 Stats
+----------
+
+We have shared our high-level community stats since 2016,
+and will continue to do it each year going forward.
+
+You can read our previous posts from 2016_, 2017_, 2018_, 2019_, 2020_, 2021_, 2022_.
+
+.. _2022: https://www.writethedocs.org/blog/write-the-docs-2022-stats/
+.. _2021: https://www.writethedocs.org/blog/write-the-docs-2021-stats/
+.. _2020: https://www.writethedocs.org/blog/write-the-docs-2020-stats/
+.. _2019: https://www.writethedocs.org/blog/write-the-docs-2019-stats/
+.. _2018: https://www.writethedocs.org/blog/write-the-docs-2018-stats/
+.. _2017: https://www.writethedocs.org/blog/write-the-docs-2017-stats/
+.. _2016: https://www.writethedocs.org/blog/write-the-docs-2016-year-in-review/
+
+Conferences
+~~~~~~~~~~~
+
+* 325 attendees in Portland (from 600 virtually)
+* 300 attendees in Atlantic (from 225)
+* 75 attendees in Australia (from 100 virtually)
+
+It was great to be back in person in 2023,
+as well as offer a new approach to our virtual events.
+
+Slack network
+~~~~~~~~~~~~~
+
+* 21,196 :doc:`members </slack>` (from 18,339)
+
+Newsletter subscribers
+~~~~~~~~~~~~~~~~~~~~~~
+
+* 11,816 :doc:`subscribers </newsletter>` (from 10,558)
+
+This is across all our various mailing lists (conference-specific and community newsletter).
+
+Meetups
+~~~~~~~
+
+* Over 10,000 :doc:`members </meetups/index>` (too many to count)
+
+Website
+~~~~~~~
+
+Unfortunately Google Analytics stopped working in 2023,
+so we don't have website stats to share.
```

Comment:
Can we switch to Plausible or something less nasty. If so we could mention it here.

Reply:
Oh dear, I didn't see anything amiss with Google Analytics! I do know that they upgraded to Analytics 4 or whatnot, but from everything I read it seems like simple websites should just migrate automatically.

Mentioning investigations of a solution to replace to reinstate it would be helpful here, I agree with Sam.

Reply:
Yea, probably makes sense to swap to plausible if we need analytics. Not 100% sure we really do though? We don't really use them for anything but this report.. 🤷